lib/rubocop/cop/rspec/message_chain.rb

```diff
@@ -0,0 +1,35 @@
+module RuboCop
+  module Cop
+    module RSpec
+      # Check that chains of messages are not being stubbed.
+      #
+      # Prefer instance doubles over stubbing any instance of a class
+      #
+      # @example
+      #   # bad
+      #   allow(foo).to receive_message_chain(:bar, :baz).and_return(42)
+      #
+      #   # better
+      #   thing = Thing.new(baz: 42)
+      #   allow(foo).to receive(bar: thing)
```

Comment:
It looks like the existing cops are using 

* bad
* good
* also good
* better
* allowed

So I’d prefer to leave my example as “better”.